bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/NMSSM_XYH_YTobb_HToWWTo2QLNu/NMSSM_XYH_YTobb_HToWWTo2QLNu_MX_mMX_MY_mMY_customizecards.dat

```diff
@@ -0,0 +1,6 @@
+set param_card mass 45 mMX
+set param_card mass 35 mMY
+set param_card mass 25 125
+set param_card DECAY 45 1.00000e-03
+set param_card DECAY 35 1.00000e-03
+set param_card DECAY 25 4.00000e-03
```

Comment:
do you have input  cross sections? what is the motivation behind the width choices? 4 e-3 vs 1e-3?

Reply:
Width for the SM-like Higgs boson (pdg Id: 25) is kept to the SM value of Higgs boson width, which is 4 MeV. For other heavy Higgs bosons (pdg Id: 35 and 45), they are taken to be of narrow width; however the choice of 1 MeV is arbitrary, but this is followed in similar X->YH analyses (ex: https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/NMSSM_XYH_bbbb/Template/customizecards.dat, https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/NMSSM_XYH_WWWW/Template/customizecards.dat). Therefore, we would also like to keep it 1 MeV so that it's easier to combine the results with the analyses pointed above.

Reply:
How do you get the cross sections for 1.e-3 width case? The cross sections you get from MadGraph isn't reliable if you don't use physics motivated widths.

Reply:
Indeed cross sections obtained from Madgraph are unphysical. So, we only use the shapes of kinematic distributions from the signal samples generated using Madgraph and to get an upper limit on the signal cross section at 95% CL. The bounds on model parameters need to be obtained from dedicated theory calculations (ex. those provided by LHC Higgs working group). This approach is also followed in the approved analyses, for example, HIG-20-014 ((X->YH.>2b2tau), B2G-21-003 (X->YH.>4b),.

Reply:
Thanks.